.rubocop.yml

```diff
@@ -3,7 +3,7 @@ inherit_gem:
 
 AllCops:
   DisplayCopNames: true
-  TargetRubyVersion: 2.5
+  TargetRubyVersion: 2.6
```

Comment:
I think that target rubocop version should be the minimum supported version of MRI

Reply:
+1